mipengine/node/tasks/views.py

```diff
@@ -24,16 +24,18 @@ def get_views(context_id: str) -> List[str]:
 
 
 @shared_task
-def create_view(
+def create_pathology_view(
     context_id: str,
     command_id: str,
     pathology: str,
     datasets: List[str],
     columns: List[str],
-    filters_json: str,
```

Comment:
On this PR there are no changes on the filters, right?

You could remove all these changes on the filters parameter to make it clear. We can fix these in the PR for the filters.